### PR TITLE
trash-cli

### PR DESCRIPTION
**Pull request by yavko**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Add trash-cli support
**What is the current behavior? (You can also link to an open issue here)**
no trash 🥺
**What is the new behavior (if this is a feature change)?**
YES TRASH 😳
**Additional info:**
5th contribution I AM COUNTING

### CODE REVIEW

**Comment by fedeci** [comment]
Can you add on the first line a comment that explains to which tool this spec refers?

e.g.
```js
// https://github.com/.../...
```

**Comment by yavko** [comment]
> Can you add on the first line a comment that explains to which tool this spec refers?
> 
> e.g.
> 
> ```js
> // https://github.com/.../...
> ```

kk

**Comment by yavko** [comment]
done

**Review by QuiiBz — APPROVED** [review]
LGTM, thanks a lot for this spec! 🎉

File: dev/trash.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        "Be verbose when moving items to the trash showing them as they are moved",
+    },
+    {
+      name: "-F",
+      description: `Ask Finder to move the files to the trash, instead of the system API. This is slower, but it utilizes Finder's (e.g. sounds) and ensures that the "put back" feature works.`,
+    },
+    {
+      name: "-l",
+      description:
+        "List items currently in the trash. If this argument is no files need to be specified",
```
Comment (fedeci) [inline dev/trash.ts:25]:
```suggestion
        "List items currently in the trash",
```

File: dev/trash.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  args: {
+    isVariadic: true,
+    template: ["folders", "filepaths"],
+  },
+
+  options: [
+    {
+      name: "-v",
+      description:
+        "Be verbose when moving items to the trash showing them as they are moved",
```
Comment (fedeci) [inline dev/trash.ts:16]:
```suggestion
        "Print verbose output while moving items",
```

File: dev/trash.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    {
+      name: "-F",
+      description: `Ask Finder to move the files to the trash, instead of the system API. This is slower, but it utilizes Finder's (e.g. sounds) and ensures that the "put back" feature works.`,
```
Comment (fedeci) [inline dev/trash.ts:20]:
```suggestion
      description: "Use the Finder API to move items to the trash, this is slower, but most MacOS features such as sounds and the \"put back\" feature work",
```

**Comment by yavko** [comment]
Ok the pr is merged but ill still make these changes on another pr!

File: dev/trash.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    {
+      name: "-s",
+      description:
+        "Securely empty the trash. trash asks for confirmation executing this action. If this argument is used, no need to be specified",
+    },
+    {
+      name: "-y",
+      description:
+        "Skips the confirmation prompt for -e and -s. CAUTION: permanently instantly",
```
[the remark this answers is not in the document]
Comment (yavko) [reply dev/trash.ts:40]: It skips the prompt for the confirmation, that's how it is in the man page

**Pull request merged**